project/project.go

```diff
@@ -40,7 +43,11 @@ func RelPPath(p string) string {
 }
 
 func BuildOut() string {
-	if os.Getenv("BUILD_OUT") != "" {
+	if *copyToRoot || Getenv("COPYTOROOT") == "true" {
```

Comment:
I'm not a real fan of this approach. I like the switch, but not two things. At some point it starts to look like xrmdb. But it's up to you.